test/component/uk/gov/hmrc/individualsincomeapi/controllers/v2/IndividualIncomeSpec.scala

```diff
@@ -18,20 +18,32 @@ package component.uk.gov.hmrc.individualsincomeapi.controllers.v2
 
 import java.util.UUID
 
-import component.uk.gov.hmrc.individualsincomeapi.stubs.{AuthStub, BaseSpec, IndividualsMatchingApiStub}
+import component.uk.gov.hmrc.individualsincomeapi.stubs.{AuthStub, BaseSpec, IfStub, IndividualsMatchingApiStub}
+import play.api.libs.json.Json
 import play.api.test.Helpers._
 import scalaj.http.Http
-import uk.gov.hmrc.individualsincomeapi.domain.SandboxIncomeData.sandboxMatchId
+import uk.gov.hmrc.individualsincomeapi.domain.integrationframework.IfPaye
+import uk.gov.hmrc.individualsincomeapi.domain.v1.SandboxIncomeData.sandboxMatchId
+import utils.IncomePayeHelpers
 
-class IndividualIncomeSpec extends BaseSpec {
+class IndividualIncomeSpec extends BaseSpec with IncomePayeHelpers {
```

Comment:
I can't view the docs on dev hub at the moment, but in employments I added checks for the error messages around fromDates / toDates at @parkeradam's suggestion -  to being before from etc. Might be worth adding to this test.